src/functions-and-arrays.js

```diff
@@ -1,21 +1,43 @@
 // Iteration #1: Find the maximum
-function maxOfTwoNumbers() {}
 
+function maxOfTwoNumbers(firstNumber,secondNumber) {
+
+  if (firstNumber > secondNumber) {
+    return firstNumber;
+  } else {
+    return secondNumber;
+  }
+}
 
 
 // Iteration #2: Find longest word
 const words = ['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot'];
 
-function findLongestWord() {}
+function findLongestWord() {
```

Comment:
in pseudocode a solution would look like this. 
init the longestWord to ""
loop over each word
if currentWord is bigger than longestWord 
update longestWord to currentWord
after loop return longestWord